packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -201,20 +203,23 @@ export class WallectConnectSession {
 
 	/**
 	 * Execute single transaction or group of transactions (atomic transaction)
-	 * @param execParams transaction parameters or atomic transaction parameters
+	 * @param transactions transaction parameters,  atomic transaction parameters
+	 *  or TransactionAndSign object(SDK transaction object and signer parameters)
 	 */
 	async executeTx(
```

Comment:
can we use `WebMode.executeTx` here?

Reply:
I think this will be use to do once all the methods return the same type